tools/disasm/files.txt

```diff
@@ -255,7 +255,7 @@
             0x80096C30 : "loadfragment2",
             0x80096C40 : "stackcheck",
             0x80096C50 : "gfxprint",
-            0x80097500 : "system_malloc",
+            0x80097500 : "",
```

Comment:
Uh.. is this an error?

Reply:
system_malloc doesn't has any data, and the data in that split seems to be referenced by another file.

Reply:
I'd rather we figured this out properly first: I don't know what an empty split name does to the disasm.

Reply:
An empty name is acceptable, it's just not obvious what .text section it's associated with if it's left empty.

Reply:
```suggestion
            0x80097500 : "boot_80086760",
```

Reply:
Are you sure that's the .text file corresponding to this .data?

Reply:
Pretty sure:
![image](https://user-images.githubusercontent.com/73679967/128782243-613e822a-0aab-4e2c-ae25-461c9147346d.png)